commands/music/music-trivia.js

```diff
@@ -155,12 +155,19 @@ module.exports = class MusicTriviaCommand extends Command {
             if (!message.guild.triviaData.triviaScore.has(msg.author.username))
               return;
             if (msg.content.startsWith(prefix)) return;
-            // if user guessed song name
-            if (msg.content.toLowerCase() === queue[0].title.toLowerCase()) {
-              if (songNameFound) return; // if song name already found
-              songNameFound = true;
 
-              if (songNameFound && songSingerFound) {
+            let guess = normalizeValue(msg.content);
+            let title = normalizeValue(queue[0].title);
+            let singer = normalizeValue(queue[0].singer);
+            
+            // if user guessed both
+            if (
+              guess.includes(singer) && guess.includes(title)
+            ) {
+              if (
```

Comment:
The if statement from line 164 checks if the user guessed both singer and title but the if statement from line 168 (which is inside the block the 'if' from 164 created) checks for one of them?

Reply:
The if statement on 164 is checking if the user's guess contains both the singer and the title. The if statement on 167 is checking if any of these has previously been guessed, so that if someone had already guessed any of those, only 1 point is attributed instead of the two.

This block is unchanged from the original code, I just moved the if clause that checks both things to the beginning and the new condition is based on .contains() and not a strict comparison. Otherwise, any of the singer/title checks would always pass the condition before the check for both.